hoomd/hpmc/shape_move.py

```diff
@@ -0,0 +1,256 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd.operation import _HOOMDBaseObject
+from . import _hpmc
+from hoomd.hpmc import integrate
+from hoomd.data.parameterdicts import ParameterDict, TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.logging import log
+import numpy
+import json
+
+
+class ShapeMove(_HOOMDBaseObject):
```

Comment:
Add a `module-hpmc-shape_move.rst` documentation file and include it in the toctree so these classes appear in the documentaiton.